extension/src/cli/git/reader.ts

```diff
@@ -68,6 +69,18 @@ export class GitReader extends GitCli {
     }
   }
 
+  public async getRemoteExperimentRefs(cwd: string): Promise<string> {
+    const options = getOptions({
```

Comment:
I think we should start doing status non-essential heavy checks in background for experiments. If we already do - what is your concern then?

Reply:
do you mean polling?

Reply:
We should look at setting up WebSockets in Studio.

Reply:
I mean that it's important that all these statutes don't affect UI. It's fine that some icons/ statuses are missing and appear in a while. WebSockets won't be enough since we need to run it the first time? (I might be missing the context of what we do here btw).